prbt_hardware_support/CMakeLists.txt

```
@@ -83,6 +83,21 @@ if(CATKIN_ENABLE_CLANG_TIDY)
 endif()
 
 
+################
+## Sanitizing ##
+################
+
+if(DEFINED ENV{ENABLE_ADDRESS_SANITIZING})
+  set (CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -g -fno-omit-frame-pointer -fsanitize=address")
```

```suggestion
  list (append CMAKE_CXX_FLAGS_DEBUG -g -fno-omit-frame-pointer -fsanitize=address")
```
should work as well. (Hold for the other "set" as well)

---

Sorry does not work. I tried it but I get some strange errors which indicate that the list is not used in the same way as the set in that case. -> Done.

---

So CMAKE_CXX_FLAGS_DEBUG does not seem to be a list but rather a string. Yep done.